docs/terraform.md

```diff
@@ -18,8 +18,8 @@ For example, a complete setup might be:
 ```
 export KOPS_STATE_STORE=s3://<somes3bucket>
 export CLUSTER_NAME=<kubernetes.mydomain.com>
-${GOPATH}/bin/kops create cluster ${NAME} --zones us-east-1c
-${GOPATH}/bin/kops update cluster ${NAME} --target=terraform
```

Comment:
Yes. This is intended to fix that inconsistency (we have `${CLUSTER_NAME}` everywhere else). On [line 20][1] we even try to set it `export CLUSTER_NAME`.

[1]: https://github.com/kubernetes/kops/pull/868/files#diff-8e2475e53e84b1e038e9f5d670c3faefR20